index.html

```diff
@@ -69,17 +66,17 @@ <h4 id='next-start'></h4>
 					<h1>Tribe Points Live!</h1>
 					<p>Updated as of: <span id="points-update-time"></span></p>
 					<br>
-					<h2><span class='myriad'>Team Gryffin</span>: <span id="points1">0</span> points</h2>
-					<h2><span class='myriad'>Team Phoenix</span>: <span id="points2">0</span> points</h2>
-					<h2><span class='myriad'>Team Fairy</span>: <span id="points3">0</span> points</h2>
-					<h2><span class='myriad'>Team Unicorn</span>: <span id="points4">0</span> points</h2>
+					<h2><span class='myriad'>Team <span class="team-name" id="name1"></span></span>: <span id="point1">0</span> points</h2>
+					<h2><span class='myriad'>Team <span class="team-name" id="name2"></span></span>: <span id="point2">0</span> points</h2>
+					<h2><span class='myriad'>Team <span class="team-name" id="name3"></span></span>: <span id="point3">0</span> points</h2>
+					<h2><span class='myriad'>Team <span class="team-name" id="name4"></span></span>: <span id="point4">0</span> points</h2>
 				</div>
 				<div id="page2bg"></div>
-				
+
 			</div>
 			<div id='page3' class='page'>
 				<div id="twitter-container" class="text-center">
-					<a class="twitter-timeline"  href="https://twitter.com/hashtag/CITYC17" data-widget-id="897168070925582339">#CITYC17 Tweets</a>
+					<a class="twitter-timeline"  href="https://twitter.com/hashtag/CITYC18" data-widget-id="897168070925582339">#CITYC18 Tweets</a>
```

Comment:
I think you might have to go to the Twitter Developer page and ask for another widget url/link for #CITYC18.